magento-research/pwa-studio
# [Documentation] Remove broken links

## Description

Remove broken links to a previously deleted topic.

## Verification Steps

0. Make sure you have `ruby` with `bundler` installed 
1. Navigate to the `pwa-devdocs` directory
2. Build the site: `npm run build`
3. Verify build succeeds with no error messages

## How Have YOU Tested this?

See **Verification Steps**

## Screenshots / Screen Captures (if appropriate):

n/a

## Checklist:
